docs/releases.md

```diff
@@ -0,0 +1,34 @@
+# Release process
+
+### High-level overview
+
+The repository is managed using [changesets](https://github.com/atlassian/changesets) workflow. It automates package versioning, changelogs. Changesets has a focus on solving these problems for multi-package repositories and keeps packages that rely on each other within the multi-package repository up-to-date, as well as making it easy to make changes to groups of packages.
+
+- Newly added changes are merged to the `development` branch first with a changeset
+- Before making a change, a pre-release mode should be made (We'll cover it later). The pre-release mode should be used only on the `development` branch
+- `changesets` GitHub action takes care of versioning and publishes packages to npm
+- When the `development` branch is ready for production release, a PR should be made to disable the pre-release mode
```

Comment:
Could you add some more information about this PR. What is going to be merged and where?

Reply:
Please see https://github.com/gnosis/safe-apps-sdk/pull/189/commits/de3162b650565823132da05081459b39255a3a97